referenceframe/frame.go

```diff
@@ -507,6 +480,7 @@ func (mf *mobile2DFrame) ProtobufFromInput(input []Input) *pb.JointPositions {
 	n := make([]float64, len(input))
 	for idx, a := range input {
 		n[idx] = a.Value
+		// TODO: should this convert deg to rad?
```

Comment:
This should convert rad to deg.

Reply:
Because all frames use degrees, correct?

Reply:
Values in Protobuf must always be in degrees.

Whether a frame wants to keep it in degrees or swap to radians for the purpose of doing math to provide its own transform is technically up to that frame, but our standard is for that to be radians.

Reply:
This structure doesn't exist anymore, going to resolve this